Corona-Warn-App/src/main/res/layout/fragment_vaccination_list.xml

```diff
@@ -0,0 +1,143 @@
+
+                    <TextView
+                        android:id="@+id/title"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_marginHorizontal="24dp"
+                        android:layout_marginTop="90dp"
+                        android:layout_marginBottom="8dp"
+                        android:gravity="center"
+                        android:text="@string/vaccination_list_title"
+                        android:textColor="@android:color/white"
+                        android:textSize="20sp"
+                        android:textStyle="bold"/>
+
+                    <TextView
+                        android:id="@+id/subtitle"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_gravity="center"
+                        android:layout_marginHorizontal="24dp"
+                        android:layout_marginBottom="12dp"
+                        android:gravity="center"
+                        android:text="@string/vaccination_list_complete_vaccination_subtitle"
+                        android:textColor="@android:color/white"
+                        android:textSize="18sp"
+                        android:visibility="visible"/>
+
+                </LinearLayout>
+
+                <com.google.android.material.appbar.MaterialToolbar
+                    android:id="@+id/toolbar"
+                    android:layout_width="match_parent"
+                    android:layout_height="?attr/actionBarSize"
+                    app:layout_collapseMode="pin"
+                    app:layout_scrollFlags="scroll|enterAlways"
+                    app:navigationIcon="@drawable/ic_close"
+                    app:navigationIconTint="@android:color/white"
+                    app:titleTextColor="@color/colorAccentTintButton">
+
+                    <LinearLayout
+                        android:id="@+id/header_text_layout"
+                        android:layout_width="match_parent"
+                        android:layout_height="match_parent"
+                        android:gravity="center_vertical"
+                        android:orientation="horizontal">
+                        <ImageView
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:layout_marginEnd="72dp"
+                            android:importantForAccessibility="no"
+                            app:srcCompat="@drawable/ic_cwa_logo_white" />
+                    </LinearLayout>
+
+                </com.google.android.material.appbar.MaterialToolbar>
+
+            </com.google.android.material.appbar.CollapsingToolbarLayout>
+
+        </com.google.android.material.appbar.AppBarLayout>
+
+        <androidx.recyclerview.widget.RecyclerView
```

Comment:
Darkmode,  `Noch 3 Tage gültig`:
Here: SecondaryTextColor/greyish color
Mockup: PrimaryTextColor/Same color as `COVID-19.....`

Reply:
Same issue in lightmode.

Reply:
Ok this seems to be different between the mockups for incomplete/complete status and light/darkmode, please check back designers what the intended text color is (primary/secondary).